week24/day01/index.js

```diff
@@ -0,0 +1,20 @@
+const countCharacMoreThanOne = str => {
+  str.toLowerCase();
+  let duplicates = [];
+  for (let index = 0; index < str.length; index++) {
+    if (str.indexOf(str[index]) !== index) {
+      if (duplicates.includes(str[index]) == false) {
+        duplicates.push(str[index]);
+      }
+    }
+  }
+  return duplicates.length;
```

Comment:
why not add each character in map and return length of keys which are greater than 2
```
let map = {}
str.forEach(char=> {
 if map[char] {
   map[char] = map[char] + 1
} else map[char] = 1
})
// filter keys which are greater than and then return length
```